DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/SimpleInjectorConfiguration.cs

```diff
@@ -21,9 +23,10 @@ public static Container RegisterPackages(Container container)
         {
             container.RegisterPackages(new[]
             {
-                typeof(DALPackage).Assembly,
-                typeof(WebApiLibraryPackage).Assembly,
-                typeof(NotesDALPackage).Assembly,
+                typeof(WebApiActivityPackage).Assembly,
+                typeof(DALSecurityPackage).Assembly,
+                typeof(WebApiSecurityPackage).Assembly,
+  				typeof(NotesDALPackage).Assembly,
```

Comment:
For fix Simple Injector in our solution create this pull request https://github.com/ITsvetkoFF/Kv-013/pull/74